couchbase/tests/common.py

```diff
@@ -0,0 +1,56 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.utils.common import get_docker_hostname
+
+HOST = get_docker_hostname()
+PORT = '8091'
+
+CHECK_NAME = 'couchbase'
+
+# Server info
+CONFIG = {
```

Comment:
no need for this anymore, let's make the single instance a fixture in `conftest.py` and kill `CONFIG`